src/main/java/ui/components/pickers/MilestonePickerState.java

```diff
@@ -105,13 +105,16 @@ private void toggleMilestone(PickerMilestone milestone) {
      */
     private void populateBestMatchingMilestones(String querySentence) {
         bestMatchingMilestones.clear();
-        addMatchingMilestones(bestMatchingMilestones, getRemainingBestMatchesToLimit(), allMilestones);
+        List<PickerMilestone> selectableMilestones = getSelectableMilestones(allMilestones);
```

Comment:
With this, should the header comment be updated? Say `... that best match querySentence and can be selected...` or something similar?